src/main/scala/org/constellation/primitives/ChannelMessage.scala

```diff
@@ -89,17 +137,26 @@ case class ChannelProof(
   checkpointMessageProof: MerkleProof
 )
 
-case class ChannelOpenRequest(
-  channelId: String,
+case class ChannelOpen(
+  name: String,
   jsonSchema: Option[String] = None,
   acceptInvalid: Boolean = true
 )
 
+case class ChannelOpenResponse(errorMessage: String = "Success", genesisHash: String = "")
```

Comment:
this is definitely weird. maybe just have an extra boolean called success, and have the default errorMessage empty?

Reply:
I just wanted a field to display on the UI for simplicity. We can change this when we add a validation spec.